iroh-sync/src/store/fs.rs

```diff
@@ -444,6 +445,13 @@ impl StoreInstance {
     fn new(namespace: NamespaceId, store: Store) -> Self {
         StoreInstance { namespace, store }
     }
+
+    fn range_start(&self) -> RecordsId {
+        (self.namespace.as_bytes(), &[0u8; 32], &[][..])
+    }
+    fn range_end(&self) -> RecordsId {
```

Comment:
these feel super hacky, if anyone has a better idea how to do this, I would love to see it. cc @rklaehn